### ISSUE TITLE
Enable test execution using a RESTful API

### ISSUE THREAD

**Issue by keshav-nangare** [opengeospatial/ets-wfs11#84]
Create an interface to execute CTL test-suite using RESTful API.

Reference: https://github.com/opengeospatial/ets-wms13/issues/34

**Comment by lgoltz** [comment]
@keshav-nangare Ist the fix for https://github.com/opengeospatial/teamengine/issues/362 required?

The name of the tes suite listed with http://localhost:8081/teamengine/rest/suites is currently  `OGC Web Feature Service 1.1.0 Exectutable Test Suite`, `Web Feature Service 1.1.0` is more readable.

The list of parameters http://localhost:8081/teamengine/rest/suites/wfs/1.33-SNAPSHOT/ described also the parameter wfs-basic which cannot be disabled. I think this can be removed from the list and described in the test.

**Comment by keshav-nangare** [comment]
@lgoltz 

Yes, teamengine fix https://github.com/opengeospatial/teamengine/issues/362  is required for all the CTL based REST interface implementation.

>  the parameter wfs-basic which cannot be disabled.

This wfs-basic CC is defined in ctl test itself thats why I have added only for information. I will remove from this list. You can see here how wfs-basic is defined: https://github.com/opengeospatial/ets-wfs11/blob/c45d5e534e183990d30a5fbe173928f0e55f8d66/src/main/scripts/ctl/main.ctl#L182
